src/Runner.Common/Util/NodeUtil.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace GitHub.Runner.Common.Util
+{
+    using System;
+    using System.Collections.ObjectModel;
+
+    public static class NodeUtil
+    {
+        public const string LatestNodeVersion = "node16";
+        public static readonly ReadOnlyCollection<string> AllowedNodeVersions = new(new[] {"node12", "node16"});
+        public static string GetForcedOrLatestNodeVersion()
+        {
+            var forcedNodeVersion = Environment.GetEnvironmentVariable(Constants.Variables.Agent.ForcedNodeVersion);
```

Comment:
should we make sure `forcedNodeVersion` is set to node12 or node16, avoid customer set `forcedNodeVersion=node100` etc..